### PR TITLE
KOGITO-5635 Added LTS native image builder

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-5635

Related PRs:
- https://github.com/kiegroup/kogito-pipelines/pull/229
- https://github.com/kiegroup/kogito-runtimes/pull/1491
- https://github.com/kiegroup/optaplanner/pull/1483
- https://github.com/kiegroup/kogito-apps/pull/973
- https://github.com/kiegroup/kogito-examples/pull/827

### CODE REVIEW

File: dsl/seed/config/branch.yaml [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
   - name: kogito-pipelines
   - name: kogito-runtimes
-  - name: kogito-apps
-  - name: optaplanner
-    jenkins_config_path: .ci/jenkins
-  - name: kogito-examples
-  - name: kogito-images
-  - name: kogito-operator
-  - name: rhpam-kogito-operator
+  # - name: kogito-apps
```
Comment (Ginxo) [inline dsl/seed/config/branch.yaml]: why don't you remove these lines? is it for testing purposes?

File: dsl/seed/config/main.yaml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 git:
   branches:
-  - main
-  - 1.9.x
+  - kogito-5627
```
Comment (Ginxo) [inline dsl/seed/config/main.yaml]: I guess this is for testing purposes... :thinking:

File: dsl/seed/jobs/seed_job_main.groovy [hunk 3]
```diff
@@ -30,8 +30,8 @@ pipelineJob('0-seed-job') {
         stringParam('CUSTOM_AUTHOR', '', 'To generate only some custom repos... Define from from which author the custom repositories are checked out. If none given, then `SEED_AUTHOR` is taken. Ignored if `CUSTOM_BRANCH_KEY` is not set.')
         stringParam('CUSTOM_MAIN_BRANCH', '', 'To generate only some custom repos... If no main_branch is given, then DSL config `git.main_branch` is taken. Ignored if `CUSTOM_BRANCH_KEY` is not set.')
 
-        stringParam('SEED_AUTHOR', 'kiegroup', 'If different from the default')
-        stringParam('SEED_BRANCH', 'main', 'If different from the default')
+        stringParam('SEED_AUTHOR', 'radtriste', 'If different from the default')
```
Comment (Ginxo) [inline dsl/seed/jobs/seed_job_main.groovy]: testing?

File: dsl/seed/config/branch.yaml [hunk 4]
```diff
@@ -35,32 +42,33 @@ repositories:
 # Main branch configuration
 git:
   author:
-    name: kiegroup
-    credentials_id: kie-ci
-    token_credentials_id: kie-ci2-token
+    name: radtriste
```
Comment (Ginxo) [inline dsl/seed/config/branch.yaml]: testing? (the same for the rest of the `radtriste` changes in this file)

File: dsl/seed/jobs/seed_job_main.groovy [hunk 5]
```diff
@@ -30,8 +30,8 @@ pipelineJob('0-seed-job') {
...
+        stringParam('SEED_BRANCH', 'test-setup', 'If different from the default')
```
Comment (Ginxo) [inline dsl/seed/jobs/seed_job_main.groovy]: testing?

**Review by Ginxo — CHANGES_REQUESTED** [review]
I'm ok with the change, but it looks more like a testing stuff than a final change...

**Comment by radtriste** [comment]
@Ginxo yeah sorry, forgot to remove the test commit ... will update native job of OP, Apps and Examples and test the jobs and then will remove. I let you know

**Review by Ginxo — APPROVED** [review]
LGTM

**Pull request merged**